packages/expo-system-ui/android/src/main/java/expo/modules/systemui/SystemUIModule.kt

```diff
@@ -0,0 +1,262 @@
+package expo.modules.systemui
+
+import android.app.Activity
+import android.content.Context
+import android.graphics.Color
+import android.os.Build
+import androidx.appcompat.app.AppCompatDelegate
+import androidx.core.view.WindowCompat
+import androidx.core.view.WindowInsetsCompat
+import androidx.core.view.WindowInsetsControllerCompat
+import expo.modules.core.ExportedModule
+import expo.modules.core.ModuleRegistry
+import expo.modules.core.Promise
+import expo.modules.core.interfaces.ActivityProvider
+import expo.modules.core.interfaces.ExpoMethod
+
+class SystemUIModule(context: Context) : ExportedModule(context) {
+
+  private lateinit var mActivityProvider: ActivityProvider
+  private val activity: Activity
+    get() {
+      return mActivityProvider.currentActivity ?: throw Error("Cannot get Main Activity!")
```

Comment:
```suggestion
      return mActivityProvider.currentActivity ?: throw Error("Cannot get Main Activity")
```
Remove all extraneous exclamation points from messages.

Reply:
`CANNOT GET MAIN ACTIVITY!! 😡🤬`